meringue-maven-plugin/src/main/java/edu/neu/ccs/prl/meringue/ReportFormat.java

```diff
@@ -0,0 +1,33 @@
+package edu.neu.ccs.prl.meringue;
+
+import org.jacoco.report.*;
+import org.jacoco.report.html.HTMLFormatter;
+import org.jacoco.report.xml.XMLFormatter;
+import org.jacoco.report.csv.CSVFormatter;
+
+import java.io.*;
+
+public enum ReportFormat {
+    HTML() {
+        @Override
+        IReportVisitor createVisitor(File outputDir) throws IOException {
+            return new HTMLFormatter().createVisitor(new FileMultiReportOutput(outputDir));
```

Comment:
Don't know what `FileMultiReportOutput` is, but presumably you got this from JaCoCo code.

Reply:
Yes, I did. This is also how Meringue originally generates an HTML report.